docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc

```diff
@@ -14,8 +14,8 @@ For previous rule updates, please navigate to the https://www.elastic.co/guide/e
 
 
 |<<prebuilt-rule-8-9-15-prebuilt-rules-8-9-15-summary, 8.9.15>> | 23 Feb 2024 | 5 | 32 | 
-updating rules for 8.9.15 release package
-
+This release includes a new rule for Windows detection of suspicious execution from INET cache. 
+Additionally, significant rule tuning for Windows and Linux rules has been added for better rule efficacy and performance.
 
```

Comment:
The update note seems to mention 1 new rule, how about the following note: "This release includes new rules for Windows including detection of suspicious execution from INET cache, potential business app masquerading, executions from removable media, XSL script using the Microsoft.XMLDOM COM interface and remote file execution via MSIEXEC. Additionally, significant rule tuning for Windows and Linux rules has been added for better rule efficacy and performance."?